.github/workflows/build-main.yml

```diff
@@ -4,30 +4,34 @@ on:
   push:
```

Comment:
using a single ci workflow, to reduce duplication.  The deploy step is the only one that's novel, and it can have a conditional to only run on specific tags/triggers

Reply:
We had originally split it up in our old cookiecutter because we wanted the `Build Main` badge to be consistently green but at this point, maintaining a single workflow seems nicer.

Reply:
elaborate?  are you saying, if tests are failing for a while it would look like the build was also failing?

Reply:
If PRs are failing it looks like the build is failing.

Reply:
hmm, I use this pattern everywhere and don't see that.  It will only look like the build is failing if you _merge_ a PR with a failed test into `main`

Reply:
Oh interesting 🤷‍♀️ maybe I had a weird config last time I tried it.